source/by-role/developer/composable/extensions/blueprints.rst

```diff
@@ -337,10 +337,40 @@ Here are some things to note in the descriptor file:
      - The license supported by the blueprint
    * - plugin.crafterCmsVersions
      - |checkmark|
-     - Contains the CrafterCMS version/s that the plugin is compatible with |br|
-       (look in the :ref:`release-notes` section for the versions available), and |br|
-       you'll need to keep this up to date
-
+     - Contains the CrafterCMS version/s that the blueprint is compatible with
```

Comment:
Start with the new system and add a note that the older system is still supported.